### DISCUSSION TITLE
About 'btc_rep_malus_civ_hd' and ACE Medical

### DISCUSSION THREAD

**Discussion by deBearN** [Vdauphin/HeartsAndMinds#1648] [Question and answer] [answered]
Does Uncon enemies count towards btc_rep_malus_civ_hd? i had this instance where during an op our reputation got really low due to btc_rep_malus_civ_hd, i modified my values so if any player do something stupid we get a huge draw back increasing Positive reputation gain and decreasing even more Negative reputation gains, currently btc_rep_malus_civ_hd gives -20 and i've just checked our .rpt and it gave one of our players over 400 -20's, i was also checking the mission via Zeus and there were roughly 4~6 civillians (which eventually got killed or turned against us), what could cause so many messages like that? all i can think of was enemies being uncon where in ACE medical they kinda turn purple via Zeus (Civ faction)

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> Does Uncon enemies count towards btc_rep_malus_civ_hd?

No

> what could cause so many messages like that?

It is using https://community.bistudio.com/wiki/Arma_3:_Event_Handlers#HandleDamage
This event is fire multiple times (up to 3-5 times per bullets), so you don't want to put a too high malus value

Cheers

**Reply by deBearN** [reply]
Understandable thank you
